jjb/install-rhel7.yaml

```diff
@@ -31,9 +31,15 @@
     builders:
       - shell: |
             #!/bin/bash -ex
+            virtualenv ansible.venv && source ansible.venv/bin/activate
+            pip install --upgrade setuptools
```

Comment:
If we change this line to the following, the installation can use Python wheel packages, which speeds up the installation significantly:

pip install --upgrade setuptools pip